test/Microsoft.Azure.WebJobs.Host.UnitTests/Retry/RetryAttributeTests.cs

```diff
@@ -62,7 +62,6 @@ public void ExponentialBackOffDelay_Constructor_Throws()
         {
             Assert.Throws<ArgumentOutOfRangeException>(() => new ExponentialBackoffRetryAttribute(5, "01:020000000000:25", "00:00:10"));
             Assert.Throws<ArgumentOutOfRangeException>(() => new ExponentialBackoffRetryAttribute(5, "01:02:25", "100000000"));
-            Assert.Throws<ArgumentException>(() => new ExponentialBackoffRetryAttribute(5, "05:02:25", "00:00:10"));
```

Comment:
intentionally removed?

Reply:
yes, the `ExponentialBackoffRetryAttribute` ctr does not throw the exception anymore.

Reply:
good to have this validation in place to avoid adding additional validation logic in the host. What do you think?

Reply:
You can retain the validation by refactoring the RandomizedExponentialBackoffStrategy ctor validation into a static Validate method on the RandomizedExponentialBackoffStrategy class that both that class ctor and ExponentialBackoffRetryAttribute ctor can call.